### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by justin-midgley**
This mostly fixes some grammar/spelling mistakes and makes the formatting a bit more consistent.
It also adds the definition of Action to the Controller API docs and fixes https://github.com/SaturnFramework/docs/issues/25

### CODE REVIEW

**Comment by iammiles** [comment]
Thanks for this. Following your updated documentation I was able to get the sample app up and running.

Is there a reason this pull request (or the others) haven't been accepted? Is there something a person like me can do to facilitate this process? Having incorrect documentation on the website seems like it would be a turn off to most folks, particularly newcomers like myself.

File: docs/api/application.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ application {
 
 ### use_jwt_authentication
 
-Enables default JWT authentication. First parameter is private key used to signing. Second one defines issuer of the token.
+Enables default JWT authentication. First parameter is private key used in signing. Second one defines issuer of the token.
```
Comment (iammiles) [inline docs/api/application.md]: Perhaps the last two sentences should be: `The first parameter is a private key used in signing. The second one defines the issuer of the token.` The extra articles make this a bit more fluid to read.

File: docs/api/application.md [hunk 2]
```diff
@@ -178,7 +178,7 @@ application {
 
 ### use_jwt_authentication_with_config
 
-Enables JWT authentication with custom configuration
+Enables JWT authentication with custom configuration.
```
Comment (iammiles) [inline docs/api/application.md]: Not really sure how I feel about some of the periods added as these aren't really complete sentences -- more headlines than anything.

File: docs/api/application.md (hunk 1 above)
Comment (justin-midgley) [reply docs/api/application.md]: Agreed. Change made in https://github.com/SaturnFramework/docs/pull/28/commits/54a5ebab9efe6edc4917bcdd7f6a5e301466fb2b

File: docs/api/application.md (hunk 2 above)
Comment (justin-midgley) [reply docs/api/application.md]: It's just for consistency. Some are sentences and therefore require periods. Others are not. I don't know how many people notice or care that some lines end in periods and some don't, but I did notice, so I thought I'd make the change.

**Comment by justin-midgley** [comment]
> Is there a reason this pull request (or the others) haven't been accepted? Is there something a person like me can do to facilitate this process? Having incorrect documentation on the website seems like it would be a turn off to most folks, particularly newcomers like myself.

I have no idea. There is no readme or other kind of info in this repo. https://github.com/SaturnFramework/Saturn#documentation welcomes help with this repo, but that's all I've found on the topic. I haven't tried to contact any maintainers directly.

**Comment by Krzysztof-Cieslak** [comment]
Thanks a lot!

**Pull request merged**